run_sil-ocpp-complete-fixture.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
Remove, seems redundant
How is this file different from run_sil-ocpp-complete.sh other than using another directory level?